microsoft-edge/privacy-whitepaper/index.md

```diff
@@ -77,6 +77,26 @@ When you visit a webpage and submit a form, Microsoft Edge sends information abo
 
 In **Guest** mode, autofill is not available and new autofill entries aren't added.  For **InPrivate** mode, Microsoft Edge offers autofill entries, but new autofill entries aren't added.
 
+<!-- ====================================================================== -->
+## Bing Chat
+
+Bing chat is an AI-enhanced web search experience. It supports users by summarizing web search results and providing a chat experience. Users can also generate creative content, such as poems, jokes, and letters. The new AI-enhanced Bing runs on a variety of advanced technologies from Microsoft and OpenAI, including GPT-4, a cutting-edge large language model (LLM) from OpenAI. 
+
+When a user enters a prompt in the new Bing, the prompt, recent conversation history, the meta prompt, page context, and top search results are sent as inputs to the LLM. The model generates a response using the page context, user’s prompt, and recent conversation history to contextualize the request, the meta prompt to align responses with Microsoft AI Principles and user expectations, and the search results to ground responses in existing, high-ranking content from the web.  
+
+Responses are presented to users in several different formats, such as traditional links to web content, AI generated summarizations, and chat responses. Summarizations and chat responses that rely on web search results will include references and a “Learn more” section below the responses, with links to search results that were used to ground the response.  
+
+Page context comprises of both page URL and the text on the webpage or local document open in the main pane. This data is only used after explicit consent from the user and only when the chat pane is open. User can close the chat pane if they want to stop the data sharing temporarily or permanently update their preferences in the Edge settings.
+
+![BingChatPane](./media/bingchat.png)
+
+Conversations on public pages are automatically saved for future reference (You can opt out of auto-saving). If conversation moves to a private page / local documents / or pages behind paywalls, conversation will not be saved to protect users’ privacy. 
```

Comment:
```suggestion
Conversations on public pages are automatically saved for future reference (you can opt out of auto-saving). If a conversation moves to a private page, local document, or a page behind a paywall, the conversation won't be saved, to protect users' privacy. 
```
* Straight quote per Docs Contrib Guide
* Lowercase in mid-sentence
* List spacing, 
* Singular usually clearer than plural